### DISCUSSION TITLE
Generators like view-component gem

### DISCUSSION THREAD

**Discussion by enderahmetyurt** [yippee-fun/phlex#26] [Ideas]
Hi,

I used before `view_component` gem that has a component generator. We can create components, and related files very quick with running something like `bin/rails generate component Example title` 

Would we add generators something like view_component has to Phlex?

Cheers.

**Comment by joeldrapper** [comment]
I really like this idea, and that's a good interface for the generator. We'll probably have to namespace the generator like this `bin/rails generate phlex:component Example title`.

**Reply by enderahmetyurt** [reply]
Great! For now I have no idea how I can implement it, but I'd like to do ☺️
